advanced/config.yaml

```diff
@@ -0,0 +1,90 @@
+components:
+  schemas:
+    AdvancedMetrics.Config:
+      description: |-
+        The container for advanced metrics configuration.
+      type: object
+      required: [choice]
+      properties:
+        choice:
+          type: string
+          enum: [events, convergence, latency]
+        events:
+          $ref: '#/components/schemas/AdvancedMetrics.Config.Events'
+        convergence:
+          $ref: '#/components/schemas/AdvancedMetrics.Config.Convergence'
+        latency:
+          $ref: '#/components/schemas/AdvancedMetrics.Config.Latency'
+
+    AdvancedMetrics.Config.Events:
+      type: object
+      properties:
```

Comment:
I feel we should provide options to selectively enable certain kinds of events. This would mean that every event type (or a set of similar event types) will have a container with at least one attr called `enable`.

We can also expose `all` to allow enabling all supported events.

Reply:
To expand more on this with examples.

```python
mcfg = api.advancedmetrics_config()
# will result in two possible events, `flow_rx_rate_above_threshold`, `flow_rx_rate_below_threshold`
mcfg.events.flow_rx_rate.enable = True
mcfg.events.flow_rx_rate.threshold_value = 95
# will result in two possible events, `port_link_up`, `port_link_down`
mcfg.events.port_link.enable = True
```